.github/workflows/post-review-ci.yml

```diff
@@ -115,3 +115,42 @@ jobs:
           export RUST_BACKTRACE=1
           ./.github/scripts/ci-test.sh
           ./.github/scripts/ci-style.sh
+
+  julia-binding-test:
+    runs-on: ubuntu-22.04
+    needs: binding-refs
+    if: contains(github.event.pull_request.labels.*.name, 'PR-testing')
+    steps:
+      - name: Checkout MMTk Core
+        uses: actions/checkout@v2
+        with:
+          path: mmtk-core
+      - name: Checkout Julia Binding
+        uses: actions/checkout@v2
+        with:
+          repository: ${{ needs.binding-refs.outputs.julia_binding_repo }}
+          path: mmtk-julia
+          ref: ${{ needs.binding-refs.outputs.julia_binding_ref }}
+      - name: Setup
+        run: |
+          cd mmtk-julia
+          ./.github/scripts/ci-checkout.sh
+          ./.github/scripts/ci-setup.sh
+          sed -i 's/^mmtk[[:space:]]=/#ci:mmtk=/g' mmtk/Cargo.toml
+          sed -i 's/^#[[:space:]]mmtk/mmtk/g' mmtk/Cargo.toml
+      - name: Overwrite MMTk core in Julia binding
+        run: |
+          mkdir -p mmtk-julia/repos/mmtk-core
+          cp -r mmtk-core/* mmtk-julia/repos/mmtk-core
```

Comment:
One alternative is to use the commands
```
cd mmtk-core
git worktree add ../mmtk-julia/repos/mmtk-core origin/master
```
to checkout another work tree.  Inside the work tree, the `git` command still considers it to be in a git repository, so our `build.rs` should still be able to encode a git SHA1 hash into the version string.

But isn't it easier to simply change `Cargo.toml` to point to `../../mmtk-core`?

Reply:
Yes. Those are all possible, and simpler. Initially, `mmtk-core` was a submodule of the binding, and we put it under `repos/mmtk-core`. But the submodule was removed, and we kept the path. In that case, we just manually copy `mmtk-core` to the path.